graylog2-web-interface/src/components/sidecars/configuration-forms/TemplatesHelper.jsx

```diff
@@ -16,31 +14,31 @@ class TemplatesHelper extends React.Component {
           </thead>
           <tbody>
             <tr>
-              <td><code>{'${'}operatingSystem{'}'}</code></td>
+              <td><code>{'${sidecar.'}operatingSystem{'}'}</code></td>
```

Comment:
Why not moving this logic into a method? In that way we would only need to replace it once if we ever change the prefix.